.github/workflows/add_issues_projects.yaml

```diff
@@ -0,0 +1,23 @@
+name: "Adiciona Issues em Projects GitHub"
+
+on:
+  workflow_call:
+    inputs:
+      project_url:
+        description: "URL do projetcs github"
+        required: true
+        type: string
+      github_token:
```

Comment:
Isso é um valor sensível, então deveria estar em [`secrets`](https://docs.github.com/pt/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets)

Reply:
Ela será utilizada como `secret`, a ideia aqui é criar um input como requirido para obrigar o usuário passar o valor dela no pipe. 
Ex. 
```
github_token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Reply:
Eu acho que ele está falando disso aqui: https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets

Que seria feito no contexto do workflow reusável e não no repo de produto.

Reply:
Boa, fiz o ajuste necessário: 
De: input
Para: secret